pubsub/azure/servicebus/servicebus.go

```diff
@@ -441,6 +450,89 @@ func (a *azureServiceBus) Subscribe(subscribeCtx context.Context, req pubsub.Sub
 	return nil
 }
 
+func (a *azureServiceBus) BulkSubscribe(subscribeCtx context.Context, req pubsub.SubscribeRequest, handler pubsub.BulkHandler) error {
```

Comment:
Could this method and `Subscribe` share more code? Perhaps having a `doSubscribe` method that can be used for both bulk and non-bulk?

Reply:
Refactored!